### ISSUE TITLE
Investigation to hide numberPad for number inputs in Welcome Setup

### ISSUE THREAD

**Issue by RiLights** [RiLights/aligntime#28]
Unfortunately I didn't find an elegant solution to hide number pad keyboard when user wants to change number by manual input. 
https://github.com/RiLights/aligntime/blob/c3b5dc39dbcc3dceb384e8446aa4e2ff7e94e535/aligntime/WelcomeSetup/DataCollect01.swift#L33

Current solutions is to show "abc" keyboard.

**Comment by OlenkaP** [comment]
@RiLights I think that abc keyboard is actually not a bad option. Can be much worse :)

**Comment by RiLights** [comment]
> Can be much worse :)
That is try :) 

Otherwise we have to make additional check for digits and letters. 
Unfortunately user have to switch from abc to numbers everytime

**Comment by OlenkaP** [comment]
@RiLights yeah from a user perspective I don’t think this is a big drama. Average person won’t even notice probably

**Comment by RiLights** [comment]
Of Course it is not a drama :)
But if we have a chance to make it work then it will be nice
